core/snippets/snippets/continue.snippet

```diff
@@ -0,0 +1,29 @@
+name: continueStatement
+phrase: continue
+insertionScope: statement
+---
+
+language: c | csharp | java
+-
+continue;
+---
+
+language: javascript | typescript | javascriptreact | typescriptreact | php | rust
+-
+continue$0;
```

Comment:
I would remove `$0` and merge this with the above definition.

Reply:
Well yes, it's just that it's seldom used and if you expect another argument is should really be a whitespace after `continue`. Let's talk about it on the community meetup.